src/components/Login/index.jsx

```diff
@@ -9,35 +9,43 @@ export default class Login extends React.Component {
     this.state = {
       isAuthenticated: false,
       isShowingError: false,
-      userName: "",
-      password: ""
+      errorMessage: "Incorrect credentials",
     };
 
     this.handleLogin = this.handleLogin.bind(this);
   }
 
   updateField(e, field) {
     this.setState({
-      [field]: e.target.value
+      [field]: e.target.value,
     });
   }
 
-  handleLogin() {
-    const { userName, password } = this.state;
+  async handleLogin() {
+    const { email, password } = this.state;
 
-    if (userName === "ziv" && password === "papa") {
-      this.setState(prevState => ({
-        isAuthenticated: !prevState.isAuthenticated
-      }));
+    const { data, status } = await authenticateUser({ email, password });
+
+    if (status === 200) {
+      if (data.id !== 0) {
+        this.setState(prevState => ({
+          isAuthenticated: !prevState.isAuthenticated,
```

Comment:
will isAuthenticated always be flipped to true if it enters this section? if so it's easier to read 
`isAuthenticated: false`

Reply:
and if there was an error, not it shouldn't display, so also add
`isShowingError: false`